tests/models/pubsub.js

```diff
@@ -718,3 +725,83 @@ addTestWithRoundedTime(
     CloseClient(client);
   }
 );
+
+Tinytest.addAsync('Models - PubSub - Waited On - track wait time of queued messages', async (test, done) => {
+  CleanTestData();
+
+  TestData.insert({aa: 10});
+  TestData.insert({aa: 20});
+
+  let client = GetMeteorClient();
+
+  const pubName = 'tinytest-waited-on';
+
+  for (let i = 0; i < 10; i++) {
+    client.subscribe(pubName);
+  }
+
+  Meteor._sleepForMs(1000);
+
+  const metrics = Kadira.models.pubsub._getMetrics(Ntp._now(), pubName);
+
+  test.isTrue(metrics.waitedOn > 1000, 'waitedOn should be greater than 1000');
+
+  done();
+});
+
+Tinytest.addAsync('Models - PubSub - Waited On - track wait time of next message', async (test, done) => {
+  CleanTestData();
+  let fastMethod = RegisterMethod(function () {
+    console.log('fastMethod');
+  });
+
+  let client = GetMeteorClient();
+
+  // If we subscribe and call before connected
+  // Meteor sends the messages in the wrong order
+  waitForConnection(client);
+
+  client.subscribe('tinytest-waited-on');
+  client.call(fastMethod, () => { });
+
+  Meteor._sleepForMs(200);
+
+  const metrics = Kadira.models.pubsub._getMetrics(Ntp._now(), 'tinytest-waited-on');
+  test.isTrue(metrics.waitedOn > 24, 'waitedOn should be greater than 24');
+
+  done();
+});
+
+Tinytest.addAsync('Models - PubSub - Waited On - track wait when unblock', async (test, done) => {
+  CleanTestData();
+  let fastMethod = RegisterMethod(function () {
+    console.log('fastMethod');
+  });
+
+  let client = GetMeteorClient();
+
+  // If we subscribe and call before connected
+  // Meteor sends the messages in the wrong order
+  waitForConnection(client);
+
+  client.subscribe('tinytest-waited-on2');
+  client.call(fastMethod, () => { });
+
+  Meteor._sleepForMs(200);
+
+  const metrics = Kadira.models.pubsub._getMetrics(Ntp._now(), 'tinytest-waited-on2');
+
+  console.log('waitedOn', metrics.waitedOn);
+
+  test.isTrue(metrics.waitedOn > 8, 'waitedOn should be greater than 8');
+  // this.unblock is provided on Meteor >= 2.3, so we expect bigger delays below this version
+
+  if (releaseParts[0] >= 2 && releaseParts[0] >= 3) {
```

Comment:
Should the second part be `releaseParts[1]`? This also will be false for Meteor 3 - 3.2.

Reply:
sure, my mistake thanks